app/models/report.py

```diff
@@ -47,3 +49,11 @@ def to_response(self) -> ReportResponse:
             score=self.score,
         )
         return response
+
+
+class ProcessedReport(Document):
+    twitter_id = StringField(primary_key=True)
+    user: dict = DictField(required=True)
+    label: int = IntField(required=True)
```

Comment:
Missing tweet information

Reply:
it was nested inside `user`

Reply:
Specify schema for `user` field instead of free form `DictField`